api/service.py

```
@@ -448,11 +448,49 @@ async def get_metrics():
+        # Determine overall status
+        all_healthy = all(
+            c.get("status") == "HEALTHY" for c in components.values()
+        )
+
+        # Get system uptime
+        uptime = time.time() - start_time
+
+        # Get current mission phase
+        try:
+            mission_phase = state_machine.get_current_phase().value
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)